build.gradle.kts

```diff
@@ -65,22 +65,16 @@ tasks {
         dependsOn(generateVersionProperties)
     }
 
-    withType<KotlinCompile>().configureEach {
+    withType<KotlinCompile> {
         kotlinOptions {
-            apiVersion = "1.4"
-            languageVersion = "1.4"
             jvmTarget = JavaVersion.VERSION_1_8.toString()
         }
     }
 
     // Required to put the Kotlin plugin on the classpath for the functional test suite
-    withType<PluginUnderTestMetadata>().configureEach {
+    withType<PluginUnderTestMetadata> {
```

Comment:
Wouldn't `withType` configure tasks eagerly? According to https://blog.gradle.org/preview-avoiding-task-configuration-time `withType().configureEach` configures tasks lazily which has positive impact on the build performance.

Reply:
You are right, reverted.